### PR TITLE
Basic file structure

### PR DESCRIPTION
**Pull request by cleverdumb**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "node src/backend/index.js"
```
Comment (Thenlie) [inline package.json:7]: Since we do not have a testing suite set up, I think we should leave this as the original text.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,2 @@
 node_modules
-.env
+.env
```
Comment (Thenlie) [inline .gitignore:2]: Would be cool to reset this file so it is not included in the PR. Not a big deal if that ends up being challenging.

**Review by Thenlie — CHANGES_REQUESTED** [review]
We may also want to hold off on file structure in general until we know the exact tech stack we are using. But not a huge deal.

File: .gitignore (hunk 2 above)
Comment (cleverdumb) [reply .gitignore:2]: not sure how to do it? i will try more tho

**Pull request closed without merging**